test_runner/batch_others/test_import.py

```diff
@@ -105,19 +105,25 @@ def import_tar(base, wal):
 
 
 @pytest.mark.timeout(600)
-def test_import_from_pageserver(test_output_dir, pg_bin, vanilla_pg, neon_env_builder):
+def test_import_from_pageserver(test_output_dir,
+                                pg_bin,
+                                vanilla_pg,
+                                neon_env_builder: NeonEnvBuilder):
+    built_type = os.environ.get("BUILD_TYPE")
+    if built_type is not None and built_type == "release":
+        log.info("Detected release build type, use a larger number of rows...")
+        num_rows = 30000000
+    else:
+        num_rows = 3000
```

Comment:
nit: These are now different tests, testing different edge cases. I'd rather be explicit about that. Maybe parameterize the test into two variants: `small` and `multisegment`, and skip `multisegment` in debug with `pytest.mark.skipif`?

Reply:
updated in https://github.com/neondatabase/neon/pull/2172/commits/fa5806a5f44f1a3076eb5a7ebf7121f5b529f576